run-in-docker.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
-# luarocks init
-# luarocks config --scope project lua_version 5.1
+#
+luarocks init
+luarocks config --scope project lua_version 5.1
+luarocks install busted 2.1.2-3
```

Comment:
I didn't realize that there would be issues with running the tests (sorry about that). This is definitely something that needs to be addressed, but I'd prefer to address it in a separate PR to keep this PR more focused. For now, could we remove the changes to this file?

Reply:
That was my intent, yes; I just left it in here so that you'd see it.

I can try setting up the tests in a slightly different way in a different PR which might be more portable.